dcm2bids/version.py

```diff
@@ -51,18 +28,19 @@ def is_tool(name):
     return which(name) is not None
 
 
-def check_github_latest(githubRepo):
+def check_github_latest(githubRepo, timeout=3):
     """ Check the latest version of a github repository
 
     Args:
         githubRepo (string): a github repository ("username/repository")
+        timeout (int): time in seconds
 
     Returns:
         A string of the version
     """
     url = "https://github.com/{}/releases/latest".format(githubRepo)
     try:
-        output = check_output(shlex.split("curl --silent " + url))
+        output = check_output(shlex.split("curl --silent " + url), timeout=timeout)
     except:
         logger.debug(
             "Checking latest version of %s was not possible", githubRepo,
```

Comment:
```suggestion
    try:
        if os.name == "nt":
            check_call(shlex.split("ping -n 1 8.8.8.8"),
                        stdout=DEVNULL,
                        stderr=STDOUT,
                        timeout=timeout)
        else:
            check_call(shlex.split("ping -c 1 8.8.8.8"),
                        stdout=DEVNULL,
                        stderr=STDOUT,
                        timeout=timeout)

    except:
        logger.info(f"Checking latest version of {githubRepo} was not possible")
        logger.debug("Can't connect to internet", exc_info=True)
        return
   try:
        output = check_output(shlex.split("curl --silent " + url), timeout=timeout)
        return (
```

Reply:
need to import 
`from subprocess import check_output, PIPE, check_call, STDOUT, DEVNULL`  as well.

Reply:
The error is the result of the following line while using the `-l DEBUG` option. It displayed the error but the code handled it.
```
logger.debug("Checking latest version of %s was not possible", githubRepo, exc_info=True)
```

When we remove the `-l DEBUG`, we don't see the error message.

I believe 'pinging' adds an extra layer and extra timeout time for no added benefit. I have opted not to add it until further feedback. However, I have split it up like you suggest to have feedback in 'normal' use cases and output the error message in debug mode:

```
logger.info(f"Checking latest version of {githubRepo} was not possible")
logger.debug("Can't connect to internet", exc_info=True)
```